### PR TITLE
Backup task capistrano

### PR DESCRIPTION
**Pull request by chuenlok**
command:  `bundle exec cap staging get_backup`
config of `local_backups_dir` is in `staging.rb`

I hope this is what you want.

### CODE REVIEW

File: lib/capistrano/tasks/backup.cap [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
   end
   
   task :pull do
     invoke 'backup:download'
-    invoke 'refresh_local'
+    invoke 'backup:refresh_local'
   end
   
   task :download do
+    backup_dir = fetch :local_backups_dir
```
Comment (ethn) [inline lib/capistrano/tasks/backup.cap:17]: does this mean we need to add local_backups_dir somewhere in our configuration?

**Comment by ethn** [comment]
good stuff.  I added a comment or two to pivotal tracker.

File: lib/capistrano/tasks/backup.cap (hunk 1 above)
Comment (chuenlok) [inline lib/capistrano/tasks/backup.cap:17]: Yes. I wrote on the first comment. 

```
config of local_backups_dir is in staging.rb
```

**Comment by chuenlok** [comment]
These lines are needed to be added to staging.rb

```
set :local_backups_dir,  '/tmp/wikirate'

set :local_wikirate_file_dir, ''
```

File: lib/capistrano/tasks/backup.cap (hunk 1 above)
Comment (ethn) [inline lib/capistrano/tasks/backup.cap:17]: doh!  sorry.  was looking at the "files changed" and missed some of the "conversation".

**Pull request merged**